change-beta/@azure-communication-react-b753ccaa-6518-4d20-b80f-4998e3f5036b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Should this be a minor change? UI update that isn't the same as before @JamesBurnside whats the rule here about this kind of change?

Reply:
Updated